ajax/libs/tootik/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "tootik",
+  "filename": "tootik.min.css",
+  "version": "1.0.2",
+  "homepage": "https://eliortabeka.github.io/tootik/",
+  "description": "A pure CSS/SCSS/LESS Tooltip library",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eliorshalev/tootik.git"
```

Comment:
`https://github.com/eliorshalev/tootik` always redirect to `https://github.com/eliortabeka/tootik`.
Let's modify to `"git+https://github.com/eliortabeka/tootik.git"`
Thanks